packages/cli/src/commands/init/index.ts

```diff
@@ -20,25 +20,11 @@ export default async function init(cwd: string) {
 
   if (fs.existsSync(changesetBase)) {
     if (!fs.existsSync(path.join(changesetBase, "config.json"))) {
-      if (fs.existsSync(path.join(changesetBase, "config.js"))) {
-        error(
-          "It looks like you're using the version 1 `.changeset/config.js` file"
-        );
-        error(
-          "The format of the config object has significantly changed in v2 as well"
-        );
-        error(
-          " - we thoroughly recommend looking at the changelog for this package for what has changed"
-        );
-        error(
-          "Changesets will write the defaults for the new config, remember to transfer your options into the new config at `.changeset/config.json`"
-        );
-      } else {
-        error("It looks like you don't have a config file");
-        info(
-          "The default config file will be written at `.changeset/config.json`"
-        );
-      }
+      error("It looks like you don't have a config file");
+      info(
+        "The default config file will be written at `.changeset/config.json`"
+      );
+
       await fs.writeFile(
```

Comment:
@asakharia 
By
> Can we have an implementation where the js extends the json ?

do you mean to create both config.json and config.js on this stepby adding something like:
```javascript
await fs.writeFile(
     path.resolve(changesetBase, "config.js"), 
     transformToJsConfig(defaultConfig)
);
```

Reply:
More like the `js` be as an extension for the `json` config. 
We can create an empty config here. 
```
await fs.writeFile(
     path.resolve(changesetBase, "config.js"), 
     transformToJsConfig({})
);
```

And allow read to extend the config

```
export const read = async (cwd: string, packages: Packages) => {
  const jsConfigPath = path.join(cwd, ".changeset", "config.js");
  const jsonConfigPath = path.join(cwd, ".changeset", "config.json");
  const jsConfigExists = await fs.pathExists(jsConfigPath);
  const jsonConfigExists = await fs.pathExists(jsonConfigPath);

  return parse({
    ...(jsonConfigExists ? await fs.readJSON(jsonConfigPath) : {})
    ...(jsConfigExists ? require(jsConfigPath) : {})
  }, packages)
};
```
Could be a cool little feature we support. Although I understand why this can be considered completely unnecessary ?
What do you think ?